tests/misc/flow-babel-only/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1,5 +1,19 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`async-arrow-invalid.js format 1`] = `
+====================================options=====================================
+parsers: ["babel"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
+<T> async () => {}
+
+=====================================output=====================================
+async <T>() => {};
+
```

Comment:
Should we throw on this?

Reply:
.... The `@babel/parser/lib/parser/error-message` is gone...

Reply:
This is expected. The published parser should be a single rollup-ed artifact. If they were published before, they are accidentally included due to incorrect publish process.